v3/src/components/codap-dnd-context.tsx

```diff
@@ -17,9 +18,10 @@ export const CodapDndContext = ({ children }: IProps) => {
                     useSensor(KeyboardSensor, { coordinateGetter: customCoordinatesGetter }),
                     // mouse sensor can be enabled for cypress tests, for instance
                     urlParams.mouseSensor !== undefined ? useMouseSensor : null)
-
+  const gridSize = 5 // pixels
+  const snapToGridModifier = createSnapModifier(gridSize)
```

Comment:
This is going to apply the snapping behavior to all drags. We only want to apply the snapping behavior to tile drags. `Modifiers` take a whole bunch of arguments that can be used to adjust the returned transform: https://github.com/clauderic/dnd-kit/blob/master/packages/core/src/modifiers/types.ts.